packages/c/crossguid/xmake.lua

```diff
@@ -0,0 +1,37 @@
+package("crossguid")
+    set_homepage("https://github.com/graeme-hill/crossguid")
+    set_description("Lightweight cross platform C++ GUID/UUID library")
+    set_license("MIT")
+
+    add_urls("https://github.com/graeme-hill/crossguid.git")
+    add_versions("master", "40aaf2e0e8fddb67dd2e1dc89091a47f1c417459b5afeeb590292fa041650952")
```

Comment:
you can add a fake version with date if you want to use master version instead of old release/tags version

```lua
    add_versions("2019.3.29", "<commit number>")
    add_versions("2019.3.29", "ca1bf4b810e2d188d04cb6286f957008ee1b7681")

```